apis/model/v1alpha1/modelversion_types.go

```diff
@@ -45,9 +45,14 @@ type ModelVersionSpec struct {
 	Storage *Storage `json:"storage,omitempty"`
 
 	// ImageRepo is the image repository to push the generated model image. e.g. docker hub.  "modelhub/mymodel"
-	// A particular image tag is automatically generated by the system. e.g.  "modelhub/mymodel:v9epgk"
+	// A particular image tag is automatically generated by the system when ImageTag is nil, e.g.
+	// "modelhub/mymodel:v9epgk"
 	// +required
 	ImageRepo string `json:"imageRepo,omitempty"`
+
+	// ImageTag is the image tag suffixed with image repo, which indicates a specific image version/variant.
```

Comment:
add: If empty, the first 5 digits of this ModelVersion object UID will be used as image tag.

Reply:
done